src/ol/object.js

```diff
@@ -197,12 +197,18 @@ ol.Object.prototype.notify = function(key, oldValue) {
  * Sets a value.
  * @param {string} key Key name.
  * @param {*} value Value.
+ * @param {boolean=} opt_silent update propertie silently
  * @api stable
  */
-ol.Object.prototype.set = function(key, value) {
-  var oldValue = this.values_[key];
-  this.values_[key] = value;
-  this.notify(key, oldValue);
+ol.Object.prototype.set = function(key, value, opt_silent) {
+  if (goog.isDef(opt_silent) && opt_silent === true) {
```

Comment:
There is no need for the `goog.isDef(opt_silent)` check here.  It should be enough to check `opt_silent === true`.